rubrik_cdm/api.py

```diff
@@ -62,6 +62,15 @@ def _common_api(self, call_type, api_version, api_endpoint, config=None,
         else:
             sys.exit('Error: "authentication" must be either True or False')
 
+        # Create required header for the special case of a bootstrap including Host attribute
+        if '/cluster/me/bootstrap' in api_endpoint:
+            header = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+                'Host': '[' + self.ipv6_addr + ']'
```

Comment:
This additional header is required when bootstrapping via link-local IPv6. Without it, the cluster returns a 400 error